widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidget.kt

```diff
@@ -40,7 +40,9 @@ internal class EntryWidgetImpl(
     override fun show(activity: Activity) = activityLauncher.launchEntryWidget(activity)
 
     override fun getView(context: Context): View {
-        val adapter = EntryWidgetAdapter(EntryWidgetContract.ViewType.EMBEDDED_VIEW, themeManager.theme?.entryWidgetTheme)
+        val adapter = EntryWidgetAdapter(
+            EntryWidgetContract.ViewType.EMBEDDED_VIEW,
+            themeManager.theme?.entryWidgetTheme)
```

Comment:
```suggestion
        val adapter = EntryWidgetAdapter(
            EntryWidgetContract.ViewType.EMBEDDED_VIEW,
            themeManager.theme?.entryWidgetTheme
        )
```
HINT: I like that style.

Reply:
Thank you. Changed. ✅